### PR TITLE
Fix #677 - Add data blocks to vendor template

### LINKED ISSUE #677: Add data blocks to vendor template
This can be done before the end of the week, and will help Laura do research on our data blocks.

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by k88hudson** [comment]
R @thisandagain or @gvn

File: lib/templates.json [hunk 1]
```diff
@@ -148,121 +148,231 @@
...
+              "id" : "innerHTML",
+              "label" : "Text Value",
+              "type" : "string",
+              "value" : "Welcome to my tea shop. What would you like to buy?"
+            }
+          },
+          "icon" : "/images/blocks_text.png",
+          "name" : "Text",
+          "type" : "text"
+        }, {
```
Comment (thisandagain) [inline lib/templates.json]: I think we do:
```
},
{
```
everywhere else.

**Comment by thisandagain** [comment]
Minor comment. R+

File: lib/templates.json (hunk 1 above)
Comment (k88hudson) [inline lib/templates.json]: true

**Comment by k88hudson** [comment]
@thisandagain ok sound be good

**Comment by thisandagain** [comment]
:+1:

**Pull request merged**